### ISSUE TITLE
ListPair within a Box doesn't align properly with other ListPairs.

### ISSUE THREAD

**Issue by coltonbrugger** [open-olive/loop-development-kit#374]
Screenshot:

<img width="412" alt="Screen Shot 2021-10-04 at 8 55 57 PM" src="https://user-images.githubusercontent.com/1859587/135948224-fc5ec228-cd33-419e-867d-16665d11e1a5.png">

Code:

```
[
   {
      "type":"listPair",
      "label":"Label Here",
      "value":"Value goes here.",
      "style":"none",
      "copyable":false,
      "layout":{
         "flex":"auto",
         "margin":null,
         "padding":null,
         "marginTop":null,
         "marginRight":null,
         "marginBottom":null,
         "marginLeft":null,
         "paddingTop":null,
         "paddingRight":null,
         "paddingBottom":null,
         "paddingLeft":null
      }
   },
   {
      "type":"box",
      "children":[
         {
            "type":"listPair",
            "label":"Label Here",
            "value":"Value goes here.",
            "style":"none",
            "copyable":false,
            "layout":{
               "flex":"auto",
               "margin":null,
               "padding":null,
               "marginTop":null,
               "marginRight":null,
               "marginBottom":null,
               "marginLeft":null,
               "paddingTop":null,
               "paddingRight":null,
               "paddingBottom":null,
               "paddingLeft":null
            }
         }
      ],
      "direction":"horizontal",
      "justifyContent":"left",
      "alignItems":"stretch",
      "layout":{
         "flex":"auto",
         "margin":null,
         "padding":null,
         "marginTop":null,
         "marginRight":null,
         "marginBottom":null,
         "marginLeft":null,
         "paddingTop":null,
         "paddingRight":null,
         "paddingBottom":null,
         "paddingLeft":null
      }
   }
]
```

**Comment by MaggieMaSS** [comment]
Hey @coltonbrugger  Thanks for the feedback! 
I have created HELPS-1709 to address this issue. We'll let you know when the problem is fixed.

**Comment by MaggieMaSS** [comment]
This issue has been fixed on Olive Helps 0.30.2.
